tests/setup.test.ts

```diff
@@ -1,6 +1,9 @@
-import { scheduleAnnounce } from "src/utils/schedule.js";
-import { describe, test } from "node:test";
+import { translateText } from "../src/translate/Translate.js";
```

Comment:
1. 
Intellisense is telling me to change this to `.../Translate.js` but when I run `jest` it only works when the path is `.../Translate`. Not sure why (config issue?)
Here's the error by intellisense:


2. 
Even when I change it to `.../Translate`, I get another import error inside `Translate.ts`: